src/pages/signin/SignInPageLayout/Footer.js

```diff
@@ -150,7 +153,7 @@ const columns = ({scrollPageToTop}) => [
 ];
 
 function Footer(props) {
-    const isVertical = props.isSmallScreenWidth;
+    const isVertical = props.isSmallScreenWidth || props.isInModal;
```

Comment:
@sobitneupane do you prefer to get value for the `isVertical` from the parent prop only and get rid of `isSmallScreenWidth` & `withWindowDimensions` here?

Reply:
@Pujan92 I think we should simply pass `shouldShowSmallScreen` from parent component and get rid of `isVertical` as a whole.

Reply:
As of now I just assigned the passed `shouldShowSmallScreen` to `isVertical` as the variable name makes more sense. Kindly review again and if you still think we need to update all occurrences of `isVertical` to `shouldShowSmallScreen`, I will do it.

Reply:
I think it's okay.